simple/stats/db.py

```diff
@@ -59,6 +59,11 @@
 
 MAIN_DC_OUTPUT_DIR = "mainDcOutputDir"
 
+_OBSERVATION_PROPERTY_COLUMNS = [
```

Comment:
Should here use the const in schema_constants.py?

Reply:
No. `schema_constants.py` has names in the KG. Here's it's the column names and in some cases they are different (e.g. `scalingFactor` vs `scaling_factor`, etc.)

Reply:
I see